Resources/doc/annotations-reference.rst

```diff
@@ -68,17 +68,29 @@ FileParam
 View
 ----
 
-.. code-block:: php
+.. configuration-block::
```

Comment:
I'm not sure if the usage of this syntax is allowed. I just took it from symfony-docs

Reply:
I started using the `::tabs` syntax like discussed in https://github.com/FriendsOfSymfony/FOSRestBundle/pull/2348#issuecomment-968292705